sdk/schemaregistry/azure-schemaregistry-avroserializer/azure/schemaregistry/serializer/avroserializer/_schema_registry_avro_serializer.py

```diff
@@ -40,14 +40,23 @@ class SchemaRegistryAvroSerializer(object):
      which is used to register schema and retrieve schema from the service.
     :type schema_registry: ~azure.schemaregistry.SchemaRegistryClient
     :param str schema_group: Schema group under which schema should be registered.
+    :keyword bool auto_register_schemas: When true, register new schemas passed to serialize.
+    Otherwise, and by default, fail if it has not been pre-registered in the registry.
```

Comment:
baby nit: I think there's a space needed
(I know the indent is some something could kill us XD)
```suggestion
     Otherwise, and by default, fail if it has not been pre-registered in the registry.
```

Reply:
good catch, updated!